main.go

```diff
@@ -191,6 +215,98 @@ func tryRebalance(ctx context.Context, r *regolancer, attempt *int) (err error,
 	return nil, true
 }
 
+func tryRapidRebalance(ctx context.Context, r *regolancer, from, to uint64, route *lnrpc.Route, amt int64) (successfullAtempts int, err error) {
+
+	rapidAttempt := 1
+
+	for {
+
+		log.Printf("Rapid Rebalance Attempt %s", hiWhiteColor(rapidAttempt))
+
+		cTo, err := r.getChanInfo(ctx, to)
+
+		if err != nil {
+			return rapidAttempt - 1, err
+		}
+		cFrom, err := r.getChanInfo(ctx, from)
+
+		if err != nil {
+			logErrorF("rapid balance error: %s", err)
```

Comment:
No idea what errors can be here but making them slightly more informative wouldn't hurt! "rapid balance error" in every "iferrnil" isn't informative enough.

Reply:
added some more verbose information